src/HealthChecks.NpgSql/NpgSqlHealthCheckOptions.cs

```diff
@@ -10,9 +10,9 @@ namespace HealthChecks.NpgSql;
 public class NpgSqlHealthCheckOptions
 {
     /// <summary>
-    /// The Postgres connection string to be used.
+    /// The Postgres data source to be used.
     /// </summary>
-    public string ConnectionString { get; set; } = null!;
```

Comment:
Removing this property will make this a breaking change.  Could you alter this PR to keep the `ConnectionString` property please?

Reply:
Ok, well thanks. Settings ConnectionString should set the DbSource